### PR TITLE
Feature/dynamic linking

### PR DESCRIPTION
**Pull request by ArminWindlin**

### CODE REVIEW

**Review by Zahikusa — APPROVED** [review]
Another very nice addition there, love it!
I can already see the orb is surpassing the user-friendliness of fandom very quickly 😄

File: src/data/articles/articleContents.json [hunk 1]
```diff
@@ -38,6 +38,18 @@
           "hideInOverview": true
         }
       ]
+    },
+    {
+      "name": "Islands",
+      "folderName": "islans",
```
[the remark this answers is not in the document]
Comment (ArminWindlin) [reply src/data/articles/articleContents.json:44]: wanna try the suggestion feature here? :)

Comment (Zahikusa) [inline src/data/articles/articleContents.json:44]:
```suggestion
      "folderName": "islands",
```

Comment (Zahikusa) [reply src/data/articles/articleContents.json:44]: Please don't tell me you made this typo on purpose just to make me try the suggestion feature :suspect: 😅

Comment (ArminWindlin) [reply src/data/articles/articleContents.json]: nice idea, but i didn't 🤣

**Pull request merged**